frontend/src/app/components/selectors/SelectorBase.ts

```diff
@@ -236,5 +236,9 @@ export abstract class SelectorBase<CustomSliceSelection extends SliceSelection>
     public deselect(): void {
         // pass
     }
+
+    public canUseMouseWheel(): boolean {
```

Comment:
Maybe this should use more descriptive name like `canUseMouseWheelToMoveBetweenSlices()`?

**Reason:** what if one of our Selectors would like to use mouse wheel, eg. zooming in/out? Then `canUseMouseWheel()` method will be confusing 😉